docs/plugin-maintainers-guide.md

```diff
@@ -4,9 +4,31 @@
 
 - [Plugin Maintainer Guide](#plugin-maintainer-guide)
   - [Table of Contents](#table-of-contents)
+  - [Plugin Owner Expectations](#plugin-owner-expectations)
+    - [PR Reviews \& Merging](#pr-reviews--merging)
+    - [Issue Triage](#issue-triage)
+  - [Version Bumping](#version-bumping)
   - [Maintaining and patching an older release line](#maintaining-and-patching-an-older-release-line)
     - [Patching an older release](#patching-an-older-release)
 
+## Plugin Owner Expectations
+
+Plugin ownership is codified by the [CODEOWNERS](https://github.com/backstage/community-plugins/blob/main/.github/CODEOWNERS) file.
+
+### PR Reviews & Merging
```

Comment:
Is there any expectation on turnaround time for initial review?

Reply:
Just my two cents, but although it would be nice to have an expectation here, we also have to keep in mind that most of the time this will be done on peoples own time 😅

Reply:
Oh, I agree with both of you. I think the addition @BethGriggs added is perfect ❤️